### PR TITLE
feat : add cads governance

### PR DESCRIPTION
**Pull request by Gab-pm**
upload governance rule for CADS

### CODE REVIEW

**Comment by fredericvilcot** [comment]
@Gab-pm @ccamel
I made some fixes to make the PR reviewable, looks good now 👍

File: example/cads/cads.ttl [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+<https://ontology.okp4.space/dataverse/zone/metadata/d0c63b88-44f8-4518-b62d-e141dd8fb624> a owl:NamedIndividual,   
+        <https://ontology.okp4.space/metadata/zone/GeneralMetadata> ;
+    core:describes zone:fdfde6f6-b5d8-4c6b-ae4c-3451b92d5314 ; 
+    core:hasDescription "Dans le cadre du projet CADS, cette zone a pour objectif de calculer un indicateur agro environnemental à partir d'un composant de l'architecture i4trust."@fr ;
+    core:hasDescription "As part of the CADS project, the aim of this zone is to calculate an agri-environmental indicator based on the i4trust architecture context broker."@en ;
+    core:hasDescription "Im Rahmen des CADS-Projekts soll in diesem Bereich ein Agrarumweltindikator auf der Grundlage des i4trust-Architekturkontextbrokers berechnet werden."@de ;
+    core:hasPublisher "OKP4" ;
+    core:hasTag "i4trust",
+        "data space";
```
Comment (fredericvilcot) [inline example/cads/cads.ttl:15]: Do we still talk about `dataspace`?

Comment (Gab-pm) [reply example/cads/cads.ttl:15]: as a tag yes, not an issue. Thanks.

**Review by fredericvilcot — COMMENTED** [review]
👍

**Review by fredericvilcot — APPROVED** [review]

**Pull request merged**